doc/en/components/charts/types/pie-chart.md

```diff
@@ -2,12 +2,27 @@
 title: $PlatformShort$ Pie Charts and Graphs | $ProductName$
 _description: The $ProductName$ pie chart is a specialized component that renders a pie chart, consisting of a circular area divided into sections.  Try for FREE.
 _keywords: $PlatformShort$ charts, pie chart, $ProductName$, Infragistics, data binding, slice selection, slice explosion, animation
-mentionedTypes: ['XamPieChart','SliceSelectionMode','PieSliceOthersContext']
+mentionedTypes: ['XamPieChart', 'XamDataChart', 'SliceSelectionMode','PieSliceOthersContext',
+'Legend',
+'LegendItemTemplate',
+'LegendItemBadgeTemplate',
+'LegendLabelMemberPath',
+'OthersCategoryThreshold',
+'OthersCategoryType',
+'SelectedItems',
+'SelectionMode',
+'RadiusFactor',
+'StartAngle',
+
+
+ ]
```

Comment:
remove property name from the mentionedTypes array. We can list only names of classes in the mentionedTypes array

Reply:
pending